### ISSUE TITLE
Map field 333 to rdau:P60327

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-resources#33]
Requested by edoweb stakeholder, example [HT014812594](http://lobid.org/resource/HT014812594).

[Source](http://lobid.org/resource?id=HT014812594&format=source) (snippet):

```xml
<datafield tag="333" ind1="-" ind2="1">
  <subfield code="a">Statistisches Landesamt Rheinland-Pfalz</subfield>
</datafield>
```

JSON:

```javascript
  "otherTitleInformation": [
    "Statistisches Landesamt Rheinland-Pfalz"
  ]
```

I am not sure whether we should do this. In other entries it seems to introduce some redundancy with field 335, though. Example [HT018166285](http://lobid.org/resource/HT018166285):

[Source](http://lobid.org/resource?id=HT018166285&format=source) (snippet):

```xml
<datafield tag="333" ind1="-" ind2="1">
  <subfield code="a">Euromedica Hannover 2010</subfield>
</datafield>
<datafield tag="335" ind1="-" ind2="1">
  <subfield code="a">Euromedica Hannover 2010 ; 3. - 4. Juni 2010 ; Internationaler medizinischer Kongress</subfield>
</datafield>
```

JSON:

```javascript
  "otherTitleInformation": [
    "Euromedica Hannover 2010 ; 3. - 4. Juni 2010 ; Internationaler medizinischer Kongress",
    "Euromedica Hannover 2010"
  ],
```

From the [MAB documentation]():

```
333       ZU ERGAENZENDE URHEBER ZUM HAUPTSACHTITEL

          Indikator:
          blank = nicht definiert


335       ZUSAETZE ZUM HAUPTSACHTITEL

          Indikator:
          blank = keine Nebeneintragung
          a     = zusaetzliche Nebeneintragung unter dem Zusatz
          b     = zusaetzliche Nebeneintragung mit dem Zusatz
```

/cc @aquast

**Comment by acka47** [comment]
@aquast and me just discussed this and we decided not to remove the field but to map it to a separate property. I think the right one is http://rdaregistry.info/Elements/u/P60327 described as "a statement associated with the title proper of a resource that relates to the identification and/or function of any agents responsible for the creation of, or contributing to the realization of, the intellectual or artistic content of the resource."

**Comment by acka47** [comment]
@jschnasse Is it ok when this is only implemented for data 2.0 or do you need it in the production data?

**Comment by acka47** [comment]
The relevant test resource for this change is HT003160768. As we still want 333 to be added as title of the resource, we will have to adjust [lines 494-501](https://github.com/hbz/lobid-resources/blob/c25daa829475b68ff80996955ee605cb51d2cf70/src/main/resources/morph-hbz01-to-lobid.xml#L494-L501):

```xml
		<combine name="@otherTitle" value="${titleRemainder}">
			<if>
				<none>
					<data source="@titleRemainder2title333"/>
				</none>
			</if>
			<data source="@titleRemainder333" name="titleRemainder"/>
		</combine>
```

When the content from 333 isn't used for the title (`@titleRemainder2title333`) it should be added to a new property http://rdaregistry.info/Elements/u/P60327 instead of being added to `@otherTitle` (http://rdaregistry.info/Elements/u/P60493). I will try to make this change after tests for #75 pass and it is merged. Maybe this morph will give the expected results then:

```xml
		<combine name="@corporateBody4Title" value="${corporateBody4Title}">
			<if>
				<none>
					<data source="@titleRemainder2title333"/>
				</none>
			</if>
			<data source="@titleRemainder333" name="corporateBody4Title"/>
		</combine>
		<data source="@corporateBody4Title" name="http://rdaregistry.info/Elements/u/P60327"/>
```

**Comment by acka47** [comment]
The changes from #77 have now to be implemented for lobid 1.0. Assigning @fsteeg.

**Comment by fsteeg** [comment]
Deployed to production, closing. See `corporateBodyForTitle` in:

http://lobid.org/resource?format=full&id=HT014812594
